lib/xcmv/group_membership.rb

```diff
@@ -13,6 +15,20 @@ def sibling_targets
       compiled_targets.select{ |t| t.source_build_phase.files_references.any?{ |f| @siblings.include?(f) } }
     end
 
+    # Returns an array of targets that the `group` should reasonably
+    # belong to -- either based on `sibling_targets` or the `sibling_targets`
+    # of some ancestor group.
+    def inferred_targets
```

Comment:
I'm curious if there's a more idiomatic way of doing this?

Reply:
Realizing there's an error above, because I conflated `GroupMembership` with `PBXGroup`. Honestly though, it might be better if `sibling_targets` is moved into `PBXGroup`.

Reply:
I added `sibling_targets` to `PBXGroup`.

Additionally, instead of checking that `group != project.main_group`, it now instead checks that `target_group` responds to `sibling_targets`, since there were situations where it would get called on `PBXProject` objects.